samples/client/petstore/dart2/petstore_client_lib/lib/api_client.dart

```diff
@@ -137,6 +137,8 @@ class ApiClient {
           return client.delete(url, headers: headerParams);
         case "PATCH":
           return client.patch(url, headers: headerParams, body: msgBody);
+        case "HEAD":
+          return client.head(url, headers: headerParams);
```

Comment:
Why is this PR updating dart2 and go-experimental files?
Does it need a rebase on master?

Reply:
Yeah at the time some samples where outdated. I have have rebased now.
I could not run `ensure_up_to_date.sh` successfully though, so not sure what’s happening there on master...